common/flatpak-dir.c

```diff
@@ -8210,6 +8210,18 @@ flatpak_dir_create_child_repo (FlatpakDir   *self,
      verify + fsync when importing to stable storage */
   ostree_repo_set_disable_fsync (repo, TRUE);
 
+  g_autofree char *cache_path = NULL;
+
+  cache_dir = flatpak_ensure_user_cache_dir_location (error);
```

Comment:
We already have a cache_dir local variable, but it is a different kind of cache dir. We want to use a separate variable for this.